snapshot.go

```diff
@@ -186,7 +186,7 @@ func getPersistentDisk(ctx context.Context, pod, namespace, prefix string) (out
 		return nil, fmt.Errorf("getting pv %q: %s", pvName, err)
 	}
 
-	if mypv.Spec.GCEPersistentDisk == nil {
+	if mypv.Spec.GCEPersistentDisk == nil && mypv.Spec.CSI == nil {
```

Comment:
I don't see any `CSI` mention in the spec on my current `pv`s, I fear that this would break here. Maybe it should be another condition?

Reply:
After reading issue #1, and re-reading the condition with fresh eyes, the condition is correct here.

But the fix is still missing something as `mypv.Spec.GCEPersistentDisk` is still used a bit lower. So it would need to change.

See https://github.com/streamingfast/snapshotter/blob/develop/snapshot.go#L211

Can you also update the error message on the condition to read maybe like:

```
no 'GCEPersistentDisk' or 'CSI' key found under Spec field of 'pv/%s' resource
```

where first argument would be `pvName`.

Reply:
CSI volume spec doesn't labels for zone & region. So creating `pdDef` is a bit tricky. I will push a workaround when I have some downtime - probably next week.